lib/utils.pm

```diff
@@ -436,15 +436,15 @@ sub workaround_type_encrypted_passphrase {
 sub ensure_unlocked_desktop {
     my $counter = 10;
     while ($counter--) {
-        assert_screen [qw(displaymanager displaymanager-password-prompt generic-desktop screenlock gnome-screenlock-password)], no_wait => 1;
+        assert_screen [qw(displaymanager displaymanager-password-prompt generic-desktop screenlock gnome-screenlock-password xscreensaver-screenlock-password)], no_wait => 1;
```

Comment:
How about we replace `gnome-screenlock-password` with `screenlock-password`? I can not imagine a situation where we need to distinguish between the two so it should be easier to reuse the more generic tag